ledfx/playlists.py

```
@@ -0,0 +1,747 @@
+                    and self._item_effective_duration_ms is not None
+                ):
+                    # Resuming: reuse previously sampled effective duration
+                    effective_duration_ms = int(
+                        self._item_effective_duration_ms
+                    )
+                else:
+                    if jitter_enabled:
+                        fmin = float(jitter.get("factor_min", 1.0))
+                        fmax = float(jitter.get("factor_max", 1.0))
+                        # sample factor uniformly
+                        factor = random.uniform(fmin, fmax)
```

Ruff (0.13.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)